doris/ddl_generic.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE bluesky (
```

Comment:
I am not an expert in Doris and Starrocks, but the corresponding DDL file for Starrocks is called `ddl_materialized.sql` (and they also have a file `ddl_default.sql`. 
- If something equivalent to `ddl_default.sql` makes sense for Doris as well, please add such a file and rename this file to `ddl_materialized.sql`.
- If something equivalent to `ddl_default.sql` makes no sense for Doris, please rename this file to `ddl.sql`.

Reply:
I add ddl_default and ddl_materialized to keep ddl same with starrocks